docs/configuration.md

```diff
@@ -23,6 +23,7 @@ Some less common environment variables are:
 Variable | Description
 ---|---
 `DISABLE_STATS` | Set to `true` to disable the the `/probot/stats` endpoint, which gathers data about each app. Recommend for apps with a lot of installations. <p>_(Example: `true`)_</p>
+`DISABLE_WEBHOOK_EVENT_CHECK` | Set to `true` to disable Probot's webhook-event-check feature. While the feature is enabled, Probot will warn you when your application is attempting to listen to an event that your GitHub App is not subscribed to. <p>_(Example: `true`)_</p>
```

Comment:
Maybe we should add a note that it defaults to `false` if `NODE_ENV` is set to `production`, otherwise `true`?